modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -79,32 +79,38 @@ Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML
 
 [source,yaml]
 ----
-outputDefaults:
- elasticsearch:
-    structuredTypeKey: openshift.labels.myLabel <1>
+outputDefaults: <1>
+  elasticsearch:
+    structuredTypeKey: kubernetes.labels.managed <2>
     structuredTypeName: nologformat
+outputs:
+  - name: elasticsearch-secure
+    elasticsearch:
+      structuredTypeKey: openshift.labels.unmanaged <3>
+      structuredTypeName: nologformat
 pipelines:
- - name: application-logs
-   inputRefs:
-   - application
-   - audit
-   outputRefs:
-   - elasticsearch-secure
-   - default
-   parse: json
-   labels:
-     myLabel: myValue <2>
+  - name: application-logs
+    inputRefs:
+    - application
+    - audit
+    outputRefs:
+    - elasticsearch-secure
+    - default
+    parse: json
+    labels:
+      unmanaged: myvalue
```

Comment:
IMO, we should add a call out and a description for this key-value pair.

Reply:
@jcantrill @anpingli Do you think we should add a callout here?

Reply:
yes,  you can. It can be any valid string.